src/controllers/redirect.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { inject } from 'inversify'
+import { controller, httpGet } from 'inversify-express-utils'
+import { RedirectResult } from 'inversify-express-utils/dts/results'
+
+import BaseController from 'app/controllers/base.controller'
+import { ApplicationStatusEnum } from 'app/models/dto/applicationStatus.enum'
+import { DissolutionGetDirector } from 'app/models/dto/dissolutionGetDirector'
+import { DissolutionGetResponse } from 'app/models/dto/dissolutionGetResponse'
+import { DissolutionApprovalModel } from 'app/models/form/dissolutionApproval.model'
+import Optional from 'app/models/optional'
+import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { ENDORSE_COMPANY_CLOSURE_CERTIFICATE_URI, REDIRECT_GATE_URI, ROOT_URI, SELECT_DIRECTOR_URI } from 'app/paths'
+import { DissolutionService } from 'app/services/dissolution/dissolution.service'
+import SessionService from 'app/services/session/session.service'
+import TYPES from 'app/types'
+
+@controller(REDIRECT_GATE_URI, TYPES.SessionMiddleware, TYPES.AuthMiddleware)
```

Comment:
don't forget TYPES.CompanyAuthMiddleware as this comes after the company auth screen